cmd/jaeger/collector-with-jaeger.yaml

```diff
@@ -0,0 +1,33 @@
+service:
+  pipelines:
+    traces:
+      receivers: [otlp, jaeger]
+      processors: [batch]
+      exporters: [kafka]
+  telemetry:
+    resource:
+      service.name: jaeger_collector
+    metrics:
+      level: detailed
+
+receivers:
+  otlp:
+    protocols:
+      grpc:
+      http:
+  jaeger:
+    protocols:
+      grpc:
+      thrift_binary:
+      thrift_compact:
+      thrift_http:
+
+processors:
+  batch:
+
+exporters:
+  kafka:
+    brokers:
+      - localhost:9092
+    topic: "jaeger-spans"
+    encoding: jaeger_json
```

Comment:
Ii feels a serious overkill to have a full config that's only different in this one line. Why don't we generate it for the test automatically by reading the main one and replacing the encoding? We already do something similar for injecting storage cleaner extension.

And if we do that, we should randomize the topic name as well (the v1 test uses random topics for each run).

Reply:
Makes sense. Will be on it again.

Reply:
I'll take up the other issue, i.e, randomizing the topic name in an other PR